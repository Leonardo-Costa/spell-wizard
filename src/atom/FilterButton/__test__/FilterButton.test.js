import React from 'react';
import renderer from 'react-test-renderer';

import {FilterButton} from '../index';

it('renders correctly', () => {
    const tree = renderer.create(<FilterButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });