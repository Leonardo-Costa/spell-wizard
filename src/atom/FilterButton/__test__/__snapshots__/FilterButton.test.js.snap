// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
    }
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "height": undefined,
        "justifyContent": "center",
        "opacity": 1,
        "width": undefined,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "height": undefined,
            "width": undefined,
          },
          Object {
            "borderRadius": 40,
          },
        ]
      }
    />
  </View>
  <Text
    style={
      Object {
        "color": "#202020",
        "fontSize": 18,
        "fontWeight": "bold",
        "marginBottom": 10,
      }
    }
  />
</View>
`;
